geonode_mapstore_client/context_processors.py

```diff
@@ -10,6 +10,18 @@
 #########################################################################
 
 from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+
+from geonode.upload.models import UploadSizeLimit
+
+
+def get_max_upload_size(slug):
```

Comment:
@marthamareal I would put this under the upload [utils.py](https://github.com/GeoNode/geonode/blob/5b1e7ae6b74658a63090a7dc68c60996542e8c8d/geonode/upload/utils.py) module

Reply:
ok, waiting for the PR on geonode